taurus/demo/cake.py

```diff
@@ -215,7 +215,7 @@ def make_cake_spec(tmpl=None):
         notes='Chocolate frosting'
     )
     IngredientSpec(
-        name="{} input".format(frosting.name),
+        name="{} input".format(frosting.name.replace('Abstract ', '')),
```

Comment:
Maybe make a utility function for these?

Reply:
Utility function for all 3 materials fields that were kwargs for the Ingredient constructors.